packages/error-debugging/dev-toolbar/src/apps/more/more-app.tsx

```
@@ -0,0 +1,85 @@
+                    <pre class="mt-3 text-[0.65rem] font-mono text-primary/70 bg-primary/5 border border-primary/15 px-3 py-2 text-left">
+{`window.__VISULIMA_DEVTOOLS__
+  .registerApp({ id, name, icon })`}</pre>
+                </div>
+            </div>
+        );
+    }
+
+    return (
+        <div class="p-5">
+            <h2 class="text-[0.65rem] font-bold uppercase tracking-[0.1em] text-muted-foreground mb-3 flex items-center gap-1.5">
+                <span aria-hidden="true" class="text-primary/50">//</span>
```

Biome (2.4.4)

[error] Wrap comments inside children within braces.

(lint/suspicious/noCommentText)